main.go

```diff
@@ -99,3 +115,50 @@ func goToolDryRunCmd(args []string) string {
 	}
 	return b.String()
 }
+
+// isGovvvDirective returns true if the arg is a govvv directive, and false
+// otherwise.
+func isGovvvDirective(arg string) bool {
+	_, ok := govvvDirectives[arg]
+	return ok
+}
+
+// scrubGovvvDirectives filters out govvv directs to return a clean set of args
+// that can be passed to the go command.
+func scrubGovvvDirectives(args []string) (filtered []string) {
```

Comment:
would be nice to have unit test for this as it's non-trivial.

Reply:
There should be 100% coverage for the new functions.